plugins/metadata/lib/kMetadataKavaUtils.php

```diff
@@ -17,6 +17,10 @@ class kMetadataKavaUtils
 	
 	protected static function getMetadataReadyFileSyncs($objectIds, $partnerId, $metadataProfileId)
 	{
+		$currentDcId = kDataCenterMgr::getCurrentDcId();
```

Comment:
I think its more nice to do:
$dcIds = kDataCenterMgr::getSharedStorageProfileIds();
$dcIds[] = kDataCenterMgr::getCurrentDcId();

Saves one line :)

Reply:
agreed ;)